src/app/common/components/modals/dialog-create-or-update-wallet/dialog-create-or-update-wallet.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import {MatDialogRef, MAT_DIALOG_DATA} from '@angular/material/dialog';
+
+@Component({
+  selector: 'app-dialog-create-or-update-wallet',
+  templateUrl: './dialog-create-or-update-wallet.component.html',
+  styleUrls: ['./dialog-create-or-update-wallet.component.scss']
+})
+export class DialogCreateOrUpdateWalletComponent implements OnInit {
+
+    constructor(
+        public dialogRef: MatDialogRef<DialogCreateOrUpdateWalletComponent>,
+        @Inject(MAT_DIALOG_DATA) public data: { title: string, btn: string, inputData: string}
```

Comment:
You should create interface for this object. Like `interface IWalletModalData`. Please also place it in different file -> you can find examples of such files in the app, those are ending with `.types.ts`